patroni/ha.py

```diff
@@ -363,6 +363,7 @@ def touch_member(self) -> bool:
                 data['tags'] = tags
             if self.state_handler.pending_restart:
                 data['pending_restart'] = True
+                data['pending_restart_reason'] = self.state_handler.pending_restart_reason
```

Comment:
Why do we use different format for the /member key and REST API?
```yaml
  "pending_restart_reason": {
    "max_connections": [
      "100",
      "101"
    ]
  }
```

vs.

```yaml
  "pending_restart_reason": {
    "max_connections": {
      "old_value": "100",
      "new_value": "101"
    }
  },
```

Reply:
the idea was to make rest api response more user-friendly, while /member key value is more for the quick access in Patroni code. But maybe it makes very little if any sense and I can make it unified

Reply:
IMO it would be better to have it unified. I'm pretty sure there are tools that could check both sources.